File: src/Example.4.Hybrid/GetPEURandomLetterCmdlet.cs
namespace PEURandom;
using System.Management.Automation;
using LoremNET;

[Cmdlet(VerbsCommon.Get, $"{Consts.ModulePrefix}Letter")]
public class GetPEURandomLetterCmdlet : PSCmdlet
{
    protected override void EndProcessing()
    {
        WriteObject(Lorem.Letter());
    }
}

File: src/Example.8.VB/GetPEURandomLetterCmdlet.vb
Imports System.Management.Automation
Imports LoremNET

<Cmdlet(VerbsCommon.Get, "PEURandomLetter")> _
Public Class GetPEURandomLetterCmdlet
   Inherits Cmdlet

   Protected Overrides Sub EndProcessing()
      WriteObject(Lorem.Letter())
   End Sub
End Class